### ISSUE TITLE
Sometimes players start with no spells. Lookup faction crashes server.

### ISSUE THREAD

**Issue by brotalnia** [AscEmu/AscEmu#422]
**Description**: When creating a new character it will randomly start with no spells, or with only some of the spells it should have. Also using the .lookup faction command crashes the server.

Video: https://www.youtube.com/watch?v=yDqR6-J4yNw

**Steps to reproduce the problem**:

1. Make a character.
2. If it has spells make another one until it bugs.

**Branch(es)**: 3.3.5

**AscEmu hash/commit**:  3b30c5e

**Operating system**:  Windows 7 x64 Ultimate

**Lua scripts**:  One to learn all weapon skills. Both issues are still present without it. http://pastebin.com/cUrtRYAH

**Other Customizations**: Instant 80.

**Comment by Zyres** [comment]
@brotalnia do you changed your optional.conf?
![opt_death](http://i.imgur.com/TR3DvUZ.png)

**Comment by brotalnia** [comment]
Yes i have changed it. But it also happens with clean config. http://pastebin.com/uSqQ1bf4
![dk clean conf](https://cloud.githubusercontent.com/assets/6519171/23823434/35edf974-066b-11e7-9c21-c1c86e727dc5.jpg)
It happens to all classes, but to death knights more often. Maybe because they start with more spells.

Also factions are not showing on the reputation window.
![no reputation](https://cloud.githubusercontent.com/assets/6519171/23823431/2696756e-066b-11e7-9247-e35bb971508d.jpg)

**Comment by Zyres** [comment]
Hmmm... I tried to get this issue but everytime everything looks fine (created 24 new chars in a row).
Can you set the following configuration in world.conf and check the console if it happen again:
![debug-world](http://i.imgur.com/Ae6Mvys.png)

**Comment by brotalnia** [comment]
I have discovered the source of the issue! Sorry for the delay. I recompiled the server and did everything from scratch on my old computer, to rule out any external factors. The bug was not there, so i copied over the new installation to my new computer and tried it there, and to my surprise the bug appeared again. Then i realized, my new computer is much faster and WoW is installed on an SSD there, so i am entering the world almost instantly after creating the character. So i waited a few seconds before entering the world with the new character, and all of my spells and reputations were there every time. If instead i spam enter and login immediately all my spells are missing. I reproduced this a dozen times and it happens every time.

Seems the server is slow in adding spells and reputations when creating the character. You have to give it about 5 seconds after clicking Accept on the character selection screen or it won't be created properly.

**Comment by brotalnia** [comment]
Here's a demonstration - https://www.youtube.com/watch?v=X6PgwKzrXt4
